### DISCUSSION TITLE
Mochi.PhysX v0.0.0-alpha0 Released!

### DISCUSSION THREAD

**Discussion by PathogenDavid** [MochiLibraries/Mochi.PhysX#7] [Announcements]
[![Mochi.PhysX v0.0.0-alpha0](https://img.shields.io/badge/Mochi.PhysX-v0.0.0--alpha0-orange?logo=nuget&style=flat-square)](https://www.nuget.org/packages/Mochi.PhysX/0.0.0-alpha0) [![Sponsor](https://img.shields.io/badge/sponsor-%E2%9D%A4-lightgrey?logo=github&style=flat-square)](https://github.com/sponsors/PathogenDavid)

See [the GitHub release](https://github.com/MochiLibraries/Mochi.PhysX/releases/tag/v0.0.0-alpha0) for details.

Special thanks to my sponsors [NeuroGEARS](https://neurogears.org/) and @AnderT. Mochi.PhysX wouldn't have ever gotten this far without your support!

**Comment by AnderT** [comment]
Hey David,

How are you going? All good I hope : )

Do you think you will be looking to upgrade to PhysX 5.1?

Cheers,

Ander

**Reply by PathogenDavid** [reply]
Hello Ander, great to hear from you!

Truth be told, not so great. Mochi Libraries lost its main source of funding recently so things are a bit of a mess right now, I'm hanging in there though!

Upgrading to PhysX 5 is something I definitely want to do, however I'm not 100% sure when I'll have time as I have quite a lot on my plate right now and it's unclear how much effort it will be. I don't think it will be too bad, but from what I saw they made a ton of breaking changes so at the very least I'm going to have to update all the samples to ensure the bindings still work.

I will let you know as soon as I have a better idea on the timeline.

Thanks for all your support,

-David
